azext_iot/sdk/digitaltwins/dataplane/azure_digital_twins_api.py

```diff
@@ -42,10 +46,11 @@ def __init__(
 
         super(AzureDigitalTwinsAPIConfiguration, self).__init__(base_url)
 
-        self.add_user_agent('azuredigitaltwinsapi/{}'.format(VERSION))
+        self.add_user_agent('azure-mgmt-device/{}'.format(VERSION))
```

Comment:
Is this a value we manually changed ourselves, or was `azuredigitaltwinsapi` the default autorest generated user-agent string?

Reply:
dammit I will change this back